File: rh-front/src/app/models/planRequest.ts
import { Employe } from "./employe";

export class PlanRequest {
    id: number;
    name: string;
    planDate: string;
    employe_id: number;

    constructor() {
        this.id = 0;
        this.name = '';
        this.planDate = '';
        this.employe_id = 0;
    }
}

File: formation-service/src/main/java/com/example/formationservice/models/PlanRequest.java
package com.example.formationservice.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.Date;

@Data @AllArgsConstructor @NoArgsConstructor
public class PlanRequest {
    private Long id;
    private String name;
    private Date planDate;
    private Long employe_id;
}
